cli/terminal/terminal.go

```diff
@@ -2,25 +2,36 @@ package terminal
 
 import (
 	"os"
+	"strconv"
+	"strings"
 
-	"github.com/buildbuddy-io/buildbuddy/cli/parser"
 	"github.com/mattn/go-isatty"
 )
 
+var (
+	// Follow bazel's behavior of only enabling color if both stdout and stderr
+	// are a tty.
+	colorEnabled = IsTTY(os.Stdout) && IsTTY(os.Stderr)
```

Comment:
Maybe add a comment that changing this value at runtime is not advised, as at least some of the calls making use of it are evaluated once and then stored without ever updating them again? Or we could use a function call to do the style for the warning/debug text, so that changing this value would be effective. (I get that it's currently supposed to be just evaluated once based on the tty-ness of the outputs and never changed, I just worry that someone might try changing it sometime in the future for some as-yet-unforeseen purpose or for testing and end up with potentially confusing or inconsistent behavior).

Reply:
Fair, done